features/datachoices/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,91 @@
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+	       xsi:schemaLocation="
+                http://www.osgi.org/xmlns/blueprint/v1.0.0
+                http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd
+                
+                http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0
+                http://aries.apache.org/schemas/blueprint-cm/blueprint-cm-1.1.0.xsd
+">
+
+    <cm:property-placeholder persistent-id="org.opennms.features.datachoices.cfg">
+        <cm:default-properties>
+            <!-- The Oracle JDK does not currently recognize the Let's Encrypt Certificate Chain
+                 in default installations (the 'ISRG Root X1' root certificate in particular)
+                 so we resort to using HTTP. -->
+            <cm:property name="url" value="http://stats.opennms.org/datachoices/"/>
```

Comment:
That URL shouldn't redirect. It's intended to be HTTP for the reason outlined in the comment.